streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/controller/AccessTokenController.java

```diff
@@ -186,6 +186,7 @@ public RestResponse copyRestApiCurl(
               .addFormData("id", appId)
               .addFormData("savePointed", "false")
               .addFormData("drain", "false")
+              .addFormData("type", "canonical")
```

Comment:
`flink/app/cancel` does not take a parameter named `type`, maybe you mean  `.addFormData("nativeFormat", "false")` here.

Reply:
yes, I have fixed it.